<launch>
    <arg name="namespace" default="ant01"/>
    <arg name="offset_x" default="0.0"/>
    <arg name="offset_y" default="0.0"/>
        <node pkg="ants_location_interface" type="antsLocationInterface" name="LocationInterface" output="screen" required="true">
            <!-- 接口话题重命名 -->
            <!-- <param name="stateEstimationTopic" type="string" value="Odometry" /> -->
            <param name="stateEstimationTopic" type="string" value="ekf/ekf_odom" />
            <param name="registeredScanTopic" type="string" value="cloud_registered" />
            <remap from="/registered_scan" to="registered_scan" />
            <remap from="/state_estimation" to="state_estimation" />
            <!-- 话题处理选项 -->
            <param name="flipStateEstimation" type="bool" value="false" />
            <param name="flipRegisteredScan" type="bool" value="false" />
            <param name="sendTF" type="bool" value="true" />
            <param name="reverseTF" type="bool" value="false" />
            <param name="NameSpace" type="string" value="$(arg namespace)" />
            <param name="offset_x" type="double" value="$(arg offset_x)" />
            <param name="offset_y" type="double" value="$(arg offset_y)" />
        </node>
</launch>